prov/gni/src/gnix_fabric.c

```diff
@@ -421,6 +444,14 @@ static int gnix_getinfo(uint32_t version, const char *node, const char *service,
 		free(gnix_info);
 	}
 
+	if (addr) {
+		free(addr);
+	}
+
+	if (str_src_addr) {
+		free(str_src_addr);
+	}
+
```

Comment:
Yes, I need to add free str_dest_addr.  This is now needed because of Chuck's rework of this function.